docs/includes/apiargs-MongoDBClient-method-addSubscriber-param.yaml

```
@@ -0,0 +1,9 @@
+arg_name: param
+name: $subscriber
+type: MongoDB\Driver\Monitoring\Subscriber
```

```suggestion
type: :php:`MongoDB\\Driver\\Monitoring\\Subscriber <class.mongodb-driver-monitoring-subscriber>`
```

See other `param.yml` files for prior art.